components/raftstore/src/store/worker/pd.rs

```diff
@@ -868,6 +960,23 @@ impl<T: PdClient + ConfigClient> Runnable<Task> for Runner<T> {
                 right_derive,
                 callback,
             ),
+            Task::AutoSplit { split_infos } => {
+                for split_info in split_infos {
+                    if let Ok(Some(region)) =
+                        self.pd_client.get_region_by_id(split_info.region_id).wait()
+                    {
+                        self.handle_ask_split(
```

Comment:
handle_ask_split? can we imporve the log such as
```
try to auto split region xxxxxxxxxxxxxxxxxx
```

Reply:
Do you mean to add a line of logs here?

Reply:
I want to know the split is caused by auto-split,